utility/src/main/java/org/oppia/util/parser/HtmlParser.kt

```diff
@@ -0,0 +1,40 @@
+package org.oppia.util.parser
+
+import android.content.Context
+import android.text.Html
+import android.text.Spannable
+import android.widget.TextView
+
+private const val CUSTOM_IMG_TAG = "oppia-noninteractive-image"
+private const val REPLACE_IMG_TAG = "img"
+private const val CUSTOM_IMG_FILE_PATH_ATTRIBUTE = "filepath-with-value"
+private const val REPLACE_IMG_FILE_PATH_ATTRIBUTE = "src"
+
+/** Html Parser to parse custom Oppia tags with Android-compatible versions. */
+class HtmlParser(private val context: Context, private val entityType: String, private val entityId: String) {
+
+  /**
+   * This method replaces custom Oppia tags with Android-compatible versions for a given raw HTML string, and returns the HTML [Spannable].
+   * @param rawString : rawString argument is the string from the string-content
```

Comment:
Nit: the ``:`` isn't needed per https://kotlinlang.org/docs/reference/kotlin-doc.html. Just having ``@param paramName param description summary`` is sufficient.

Reply:
corrected.